units/systemd-time-wait-sync.service.in

```diff
@@ -22,8 +22,8 @@ ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 
 DefaultDependencies=no
-Before=time-sync.target shutdown.target
-Wants=time-sync.target
+Before=time-set.target time-sync.target shutdown.target
+Wants=time-set.target time-sync.target
```

Comment:
hmm, why this change? this service waits for the ntp sync hence should only be order before classic time-sync.target

Reply:
At the time I thought it necessary to get the synchronization points.  Today it doesn't, so I've 
removed this change.